GenericDiffractometer.py

```diff
@@ -117,14 +117,27 @@ class GenericDiffractometer(HardwareObject):
                    "kappa",
                    "kappa_phi",
                    "beam_x",
-                   "beam_y"]
+                   "beam_y",
+                   "back_light",
```

Comment:
Since you had zoom, beam_x/y…. I had thought it would be the diffractometer motors in general. That’s why I created another centring motor list for centring, self.centring_motors_list = ["phiz", "phiy","sampx","sampy"]


From: Ivars Karpics [mailto:notifications@github.com]
Sent: Thursday, April 14, 2016 3:16 PM
To: mxcube/HardwareObjects
Cc: Jie Nan
Subject: Re: [mxcube/HardwareObjects] Improvement of Genericdiffractometer (#31)


In GenericDiffractometer.py<https://github.com/mxcube/HardwareObjects/pull/31#discussion_r59714130>:

> @@ -117,14 +117,27 @@ class GenericDiffractometer(HardwareObject):

>                     "kappa",

>                     "kappa_phi",

>                     "beam_x",

> -                   "beam_y"]

> +                   "beam_y",

> +                   "back_light",

Hi @JieNanMAXIV<https://github.com/JieNanMAXIV>

Sorry for disturbing, but MOTORS_NAME is a list to be used for centering positions (to create, compare, etc.). I think back_light and front_light should not be here.
What do you think?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/mxcube/HardwareObjects/pull/31/files/df567d5a90b84782670a548677e8561685fe5adc#r59714130>